test/format/config.c

```diff
@@ -358,6 +375,25 @@ config_run(void)
 {
     config_random(tables[0], false); /* Configure the remaining global name space. */
 
+    /*
+     * Limit the number of tables to 5 if realloc exact and realloc malloc are both on and not all
```

Comment:
I'd get rid of the actual number in the comment and make the use below a macro like REALLOC_MAX_TABLES.
That way if we change it, this comment can just be something that fits all values:
/* Limit the number of tables if realloc...

Reply:
done